scylla-overview: Add SCT test support

k0machi opened the pull request:
This commit adds SCT metrics onto the Overview dashboard, migrating them
from the old dump stored in the SCT repo and instead putting them into
the template with optional "sct_tests" dashproduct flag, allowing them
to be conditionally added to the dashboard.

Task: https://github.com/scylladb/qa-tasks/issues/1444
Closes #2419

amnonh commented:
Putting the entire JSON dump inside a template file defeats the purpose of a template file.
Do you want it as a separate dashboard? or do you want it to override the overview dashboard?

amnonh committed:
scylla-overview: Add SCT test support

This is a work in progress to place sct related panels inside the
overview dashboard.

This is an example of how to generate the dashboard for 6.1 release:

./generate-dashboards.sh -F -v 6.1 -P "sct-tests"

Signed-off-by: Amnon Heiman <amnon@scylladb.com>

k0machi committed:
scylla-overview: Add remaining SCT metrics

This commit adds SCT metrics onto the Overview dashboard, migrating them
from the old dump stored in the SCT repo and instead putting them into
the template with optional "sct_tests" dashproduct flag, allowing them
to be conditionally added to the dashboard.

Task: scylladb/qa-tasks#1444

k0machi commented:
Still somewhat of a WIP but metrics have been ported:
![image](https://github.com/user-attachments/assets/6f3bb895-d016-4bca-88b0-d91116589d8d)

k0machi committed:
scylla-overview: Remove unnecessary metrics and adjust rows

Removed duplicate metrics available on other dashboards and also
adjusted the layout and some of the panels unit/type configuration

amnonh commented on grafana/scylla-overview.template.json:
```diff
@@ -8,125 +8,1193 @@
+                        "title": "Requests Served per [[by]]",
+                        "type": "timeseries"
+                    },
+                    {
```
I prefer to move this part to types.json i.e. create a new class for annotation list

amnonh commented on grafana/scylla-overview.template.json:
```diff
@@ -8,125 +8,1193 @@
+                                "interval": "15s",
+                                "intervalFactor": 1,
+                                "legendFormat": "YCSB READ [{{instance}}]",
+                                "refId": "G"
```
why the frefid not starting from A?

amnonh commented:
The PR includes formatting over the entire document. It first combines tabs and spaces, and even without them, you should do it in a different patch.

Besides the specific type that will be best put in types.json, as it's a lot of rows that are  only relevant to QA, you can define the the few rows inside types.json so the amount of unrelated code inside the overview file will be limited (look at small_stats_row for a reference)
